tests/app/main/test_permissions.py

```diff
@@ -373,6 +373,45 @@ def test_service_navigation_for_org_user(
     assert [
         item.text.strip() for item in page.select('nav.navigation a')
     ] == [
+        'Usage',
+        'Team members',
+    ]
+
+
+def test_org_users_can_always_see_usage_page(
```

Comment:
I think it’s good to call out that this is explicitly about the usage page, since it’s the only special one.
```suggestion
def test_org_user_who_is_also_service_user_can_still_see_usage_page(
```